apps/admin/src/styles.css

```
@@ -0,0 +1,1346 @@
+  margin-top: 3px;
+  color: #697066;
+  font-size: 12px;
+}
+
+.admin-json-preview {
+  max-height: 260px;
+  overflow: auto;
+  margin: 0;
+  background: #f8f7f1;
+  color: #2d332b;
+  font-family: "Cascadia Mono", "SFMono-Regular", Consolas, monospace;
```

Stylelint (17.11.1)

[error] Expected no quotes around "SFMono-Regular" (font-family-name-quotes)

(font-family-name-quotes)